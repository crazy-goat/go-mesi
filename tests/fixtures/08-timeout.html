<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
Large timeout: [<esi:include timeout="0.2" src="http://127.0.0.1:8080/sleep/1/2">Timeout</esi:include>]
Close timeout: [<esi:include timeout="0.9999" src="http://127.0.0.1:8080/sleep/1/1">Timeout</esi:include>]
No timeout: [<esi:include timeout="1.2" src="http://127.0.0.1:8080/sleep/1/3">Timeout</esi:include>]
No timeout on first error: [<esi:include timeout="1.2" src="http://127.0.0.1:8080/status/code/404" alt="http://127.0.0.1:8080/sleep/1/4">Timeout</esi:include>]
Alt timeout: [<esi:include timeout="1.2" src="http://127.0.0.1:8080/sleep/2/5" alt="http://127.0.0.1:8080/sleep/1/6">Timeout</esi:include>]
No alt on timeout: [<esi:include timeout="0.2" src="http://127.0.0.1:8080/sleep/1/7" alt="http://127.0.0.1:8080/hello">Timeout</esi:include>]
</body>
</html>
