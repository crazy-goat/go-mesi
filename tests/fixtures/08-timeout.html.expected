<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
Large timeout: [Timeout]
Close timeout: [Timeout]
No timeout: [3 Waited 1]
No timeout on first error: [4 Waited 1]
Alt timeout: [Timeout]
No alt on timeout: [Timeout]
</body>
</html>
